app/components/containers/HTTPError.js

```diff
@@ -1,5 +1,11 @@
+/* eslint-disable global-require */
 import React from 'react';
 
-const HTTPError = ({ code }) => (<h1>Error {code}</h1>);
+const HTTPError = ({ code }) => (
+  <div>
```

Comment:
css is applied to `main` so using it will mess with my stuff.  Discuss possible format for css?  Or I'll make a class to override this.